snuba/migrations/operations.py

```diff
@@ -124,6 +124,44 @@ def format_sql(self) -> str:
         return f"TRUNCATE TABLE IF EXISTS {self.__table_name};"
 
 
+class ModifyTableTTL(SqlOperation):
```

Comment:
Why not adding an attribute to this class to decide whether to materialize the TTL and force the parameter to that value for the session the migration is executed in ?

Reply:
That's a good suggestion. I have modified the code to add that setting when running the command.